seldon-core-operator/tox.ini

```
@@ -0,0 +1,74 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+skipsdist = True
+skip_missing_interpreters = True
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}
+    PYTHONBREAKPOINT=ipdb.set_trace
+
+[testenv:prepare-rock]
+allowlist_externals =
+    bash
+    git
+    skopeo
+    tox
+    rm
+    rockcraft
+    yq
+setenv =
+    # Needed for juju cli to work correctly
+    HOME={env:HOME}
+    CHARM_REPO=https://github.com/canonical/seldon-core-operator.git
+    CHARM_BRANCH=main
+    LOCAL_CHARM_REPO=charm_repo
+deps =
+    juju~=3.0.0
```

See [this discussion](https://github.com/canonical/istio-operators/pull/272#issuecomment-1560056469)
```suggestion
    juju~=2.9.0
```